include/compare_main.h

```diff
@@ -29,11 +29,9 @@ using std::set;
 using std::vector;
 
 struct CompareOptions {
-    fs::path prgfile;
+    fs::path pandora_index_file;
```

Comment:
Maybe just simpler as `index_file`? It's kind of obvious it's a pandora index no? Anyway, that's a very minor thing. Don't worry about changing it if you can't be bothered

Reply:
Done in https://github.com/rmcolq/pandora/pull/318/commits/2631de2697d65def8622931117686db46f6b213b